protos/camera/camera.proto

```diff
@@ -57,34 +53,57 @@ service CameraService {
      * List photos available on the camera.
      */
     rpc ListPhotos(ListPhotosRequest) returns(ListPhotosResponse) {}
+    /*
+     * Subscribe to list of cameras.
+     *
+     * This allows to find out what cameras are connected to the system.
+     * Based on the camera ID, we can then address a specific camera.
+     */
+    rpc SubscribeCameraList(SubscribeCameraListRequest) returns(stream CameraListResponse) {}
     /*
      * Subscribe to camera mode updates.
      */
-    rpc SubscribeMode(SubscribeModeRequest) returns(stream ModeResponse) {}
+    rpc SubscribeMode(SubscribeModeRequest) returns(stream ModeResponse) { option (mavsdk.options.async_type) = ASYNC; }
```

Comment:
I am not sure I understand the change putting all the streams as ASYNC and adding a Get* version for all of them. Isn't it already the case that the SYNC version is blocking (but it is called `mode()` instead of `get_mode()`)? E.g. https://github.com/mavlink/MAVSDK/blob/main/src/mavsdk/plugins/camera/include/plugins/camera/camera.h#L675

Reply:
So there is a catch with multiple cameras.

If you leave it the way it is, you get subscriptions for all cameras, as they get updated. However, when you call `mode()` which one do you get? To avoid that, I remove `mode()` and add `get_mode(camera_id)`.

Reply:
Oh, of course!